rng.h

```diff
@@ -0,0 +1,32 @@
+#include <random>
+
+class Random
+{
+public:
+    bool BinaryEvent(float probability = 0)
```

Comment:
Could make this a one liner:
```suggestion
    bool BinaryEvent(float prob = 0) { return ApplyBernoulli(probability); };
```